app/controllers/manage_collections_controller.rb

```diff
@@ -1,12 +1,11 @@
 # :nodoc:
 class ManageCollectionsController < PagesController
   include BulkUpdates
-  include ManageCollectionsHelper
 
-  layout 'manage_collections'
+  layout 'manage_metadata'
```

Comment:
I think the plan was to keep the Manage Metadata controller empty, but also having a manage collections, variables and services controller. Although, we would only have one layout, the manage metadata layout, to accommodate all of their views. May need @macrouch or @abbottry to verify that, but that is what is in MMT-1033.